src/index.html

```diff
@@ -1,159 +1,169 @@
+          </section>
+          <section class="sleep-button current-stats">
+            <div class="daily-stat-icons">
+              <img src="../assets/sleep.svg" class="sleep-icon icon" alt="icon of person sleeping">
+              <p class="stats-text">sleep</p>
+            </div>
+          </section>
+          <section class="hydration current-stats">
+            <div class="daily-stat-icons">
+              <img src="../assets/water.svg" class="water-icon icon" alt="icon of water droplet">
+              <p class="stats-text">water</p>
+            </div>
+          </section>
+        </article>
+      </section>
+      <section class="user-stats">
+          <div class="activity-stats reset">
+            <h2>Today's Activity</h2>
+            <p class="today-activity"></p>
+            <p class="avg-activity-stats"></p>          
+            <p class="stairs-compare"></p>  
+            <p class="steps-compare"></p>
+            <p class="minutes-compare"></p>
+            <table class="activity-data table">
+              <th class="weekly-activity">Start date</th>
+              <td class="activity today-activity"></td>
+              <tr>
+                <th class="weekly-activity">Next day</th>
+                <td class="activity yesterday-activity"></td>
+              </tr>       
+              <tr>
+                <th class="weekly-activity">2 days later</th>
+                <td class="activity two-day-activity"></td>
+              </tr>
+              <tr>
+                <th class="weekly-activity">3 days later</th>
+                <td class="activity three-day-activity"></td>
+              </tr>
+              <tr>
+                <th class="weekly-activity">4 days later</th>
+                <td class="activity four-day-activity"></td>
+              </tr>
+              <tr>
+                <th class="weekly-activity">5 days later</th>
+                <td class="activity five-day-activity"></td>
+              </tr>
+              <tr>
+                <th class="weekly-activity">6 days later</th>
+                <td class="activity six-day-activity"></td>
+              </tr>
+            </table>
           </div>
-        </section>
-      </article>
-    </section>
-
-    <section class="user-stats">
-      <!-- <h2>My Activity</h2> -->
-      <article class="display-stats">
-        <div class="activity-stats reset">
-          <h2>Today's Activity</h2>
-          <p class="today-activity"></p>
-          <div class="avg-activity-stats"></div>          
-          <p class="stairs-compare"></p>  
-          <p class="steps-compare"></p>
-          <p class="minutes-compare"></p>
-          <table class="activity-data table">
+          <div class="sleep-stats reset hidden">
+            <h2>Today's Sleep</h2>
+            <p class="today-sleep"></p>
+            <p class="avg-sleep-stats"></p>
+            <table class="sleep-data table">
+              <th class="weekly-sleep">Start date</th>
```

Comment:
Do these need to be in a `tr`?

Reply:
Added a tr around all these first rows in the tables